server/src/main/java/org/apache/iotdb/db/metadata/MTree.java

```diff
@@ -212,6 +212,11 @@ MeasurementMNode createTimeseries(PartialPath path, TSDataType dataType, TSEncod
       }
       cur = cur.getChild(nodeName);
     }
+
+    if (props != null && props.containsKey("loss") && props.get("loss").equals("sdt")) {
```

Comment:
Hi, something like "sdt" or "loss" should not be hard-code. It's better to extract them to be a variable.